### PR TITLE
missing line for single docs

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

**Review by RickWinter — APPROVED** [review]

File: sdk/core/azure-core/README.md [hunk 1]
```diff
@@ -50,11 +50,12 @@ auto finalResult = operation.Value();
 
 ### HTTP Transport adapter
 
-Out of the box, the Azure SDK for C++ supports the libcurl and WinHTTP libraries as HTTP stacks for communicating with Azure services over the network. The SDK also provides a mechanism for `customer-implemented` *HTTP transport adapter*. [You can learn more about the transport adapter in this doc](https://github.com/Azure/azure-sdk-for-cpp/blob/master/doc/HttpTransportAdapter.md#http-transport-adapter).
+Out of the box, the Azure SDK for C++ supports the libcurl and WinHTTP libraries as HTTP stacks for communicating with Azure services over the network. The SDK also provides a mechanism for `customer-implemented` _HTTP transport adapter_. [You can learn more about the transport adapter in this doc](https://github.com/Azure/azure-sdk-for-cpp/blob/master/doc/HttpTransportAdapter.md#http-transport-adapter).
```
Comment (ahsonkhan) [inline sdk/core/azure-core/README.md:53]: Why the bold to italics change?

File: sdk/core/azure-core/README.md [hunk 2]
```diff
@@ -78,16 +80,17 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
...
+### Additional Helpful Links for Contributors
 
-Many people all over the world have helped make this project better.  You'll want to check out:
+Many people all over the world have helped make this project better. You'll want to check out:
 
-* [What are some good first issues for new contributors to the repo?](https://github.com/azure/azure-sdk-for-cpp/issues?q=is%3Aopen+is%3Aissue+label%3A%22up+for+grabs%22)
-* [How to build and test your change][azure_sdk_for_cpp_contributing_developer_guide]
-* [How you can make a change happen!][azure_sdk_for_cpp_contributing_pull_requests]
-* Frequently Asked Questions (FAQ) and Conceptual Topics in the detailed [Azure SDK for C++ wiki](https://github.com/azure/azure-sdk-for-cpp/wiki).
+- [What are some good first issues for new contributors to the repo?](https://github.com/azure/azure-sdk-for-cpp/issues?q=is%3Aopen+is%3Aissue+label%3A%22up+for+grabs%22)
```
Comment (ahsonkhan) [inline sdk/core/azure-core/README.md:87]: 👍

File: sdk/core/azure-core/inc/azure/core/operation.hpp [hunk 3]
```diff
@@ -105,6 +105,7 @@ namespace Azure { namespace Core {
 
     /**
      * @brief Returns the current #Azure::Core::OperationStatus of the long-running operation.
+     *
```
Comment (ahsonkhan) [inline sdk/core/azure-core/inc/azure/core/operation.hpp:108]: Out of curiosity, what regex did you use?

File: sdk/keyvault/azure-security-keyvault-keys/src/private/package_version.hpp [hunk 4]
```diff
@@ -20,6 +20,7 @@
 namespace Azure { namespace Security { namespace KeyVault { namespace Keys { namespace _detail {
   /**
    * @brief Provides version information.
+   *
```
Comment (ahsonkhan) [inline sdk/keyvault/azure-security-keyvault-keys/src/private/package_version.hpp:23]: I am guessing storage needs to make a similar change? Please share what you did to make it easier.

cc @Jinming-Hu

**Comment by ahsonkhan** [comment]
Part of https://github.com/Azure/azure-sdk-for-cpp/issues/2260

File: sdk/core/azure-core/README.md (hunk 1 above)
Comment (vhvb1989) [reply sdk/core/azure-core/README.md:53]: I think bold would require **  isn't it?   I got all this file auto-formated based on the https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-markdown VSCode extension, I believe

File: sdk/core/azure-core/inc/azure/core/operation.hpp (hunk 3 above)
Comment (vhvb1989) [reply sdk/core/azure-core/inc/azure/core/operation.hpp:108]: `/\*\*[\n\s]+.*[\n\s]+\*/`
/\*\*[\n\s]  ->  finds `/** then-whatever up to end of line
+ -> adds multiline support to check next line
.*  -> whatever on this line up to end of line
*/  ->  end of doxygen..

So, any doxygen comment with one line length only :)

File: sdk/keyvault/azure-security-keyvault-keys/src/private/package_version.hpp (hunk 4 above)
Comment (vhvb1989) [reply sdk/keyvault/azure-security-keyvault-keys/src/private/package_version.hpp:23]: use regex `/\*\*[\n\s]+.*[\n\s]+\*/` to find all cases, then add one line :)

**Pull request merged**